.github/stale.yaml

```diff
@@ -0,0 +1,18 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
```

Comment:
Is this section used now? If not please comment it out.

Reply:
@Madhu-1 its a place holder and we could use it

Reply:
Is there a need to use it? Maybe is it useful for the the do-not-merge-label?